store/memory.go

```diff
@@ -134,7 +134,7 @@ func (e *MemStore) Get(_ context.Context, commit []byte) ([]byte, error) {
 func (e *MemStore) Put(_ context.Context, value []byte) ([]byte, error) {
 	time.Sleep(e.config.PutLatency)
 	if uint64(len(value)) > e.config.MaxBlobSizeBytes {
-		return nil, fmt.Errorf("blob is larger than max blob size: blob length %d, max blob size %d", len(value), e.config.MaxBlobSizeBytes)
+		return nil, fmt.Errorf("%s: blob length %d, max blob size %d", OversizedProxyError, len(value), e.config.MaxBlobSizeBytes)
```

Comment:
```suggestion
		return nil, fmt.Errorf("%w: blob length %d, max blob size %d", OversizedProxyError, len(value), e.config.MaxBlobSizeBytes)
```

Reply:
wrap errors instead of just concatenating strings. Will require making OverSizedProxyError an actual error, but this way we can use golang's stdlib's error functionality.

Reply:
[97c51a0](https://github.com/Layr-Labs/eigenda-proxy/pull/140/commits/97c51a0994e1a02077ce6f6a432a1ff6e44d8fd4)